src/features/node-common.js

```diff
@@ -0,0 +1,49 @@
+import fs from 'fs';
+import _path from 'path';
+import url from 'url';
+
+import fileUrl from 'file-url';
+
+
+export const URL = global.URL
+  ? global.URL
+  : url.URL;
+
+
+export const pathToFileURL = url.pathToFileURL
+  ? url.pathToFileURL
+  : function pathToFileURL(path) {
+    const theUrl = new URL(fileUrl(path));
+    if (path.endsWith(_path.sep)) {
+      theUrl.pathname += '/';
+    }
+    return theUrl;
+  };
+
+export const DEFAULT_BASEURL = pathToFileURL(process.cwd() + '/');
```

Comment:
This should probably be moved to `common.js` so that it can handle both browser and Node environments.